lispyville.el

```diff
@@ -1890,7 +1958,10 @@ When THEME is not given, `lispville-key-theme' will be used instead."
            [remap evil-delete-backward-char]
            #'lispyville-delete-char-or-splice-backwards
            [remap evil-substitute] #'lispyville-substitute
-           [remap evil-change-whole-line] #'lispyville-change-whole-line))
+           [remap evil-change-whole-line] #'lispyville-change-whole-line
+           [remap evil-join] #'lispyville-join)
+         (lispyville--define-key 'normal
```

Comment:
Remapping `evil-join` is fine, but I don't want to override a user's binding for `gc` by default. Please remove the keybinding or put it in a separate `comment` key theme.